modules/effects/schematics-core/utility/ast-utils.ts

```diff
@@ -636,3 +643,52 @@ export function insertImport(
     ts.SyntaxKind.StringLiteral
   );
 }
+
+export function replaceImport(
+  sourceFile: ts.SourceFile,
+  path: Path,
+  importFrom: string,
+  importAsIs: string,
+  importToBe: string
+): ReplaceChange[] {
+  const imports = sourceFile.statements
+    .filter(ts.isImportDeclaration)
+    .filter(
+      ({ moduleSpecifier }) =>
+        moduleSpecifier.getText(sourceFile) === `'${importFrom}'` ||
+        moduleSpecifier.getText(sourceFile) === `"${importFrom}"`
+    );
+
+  if (imports.length === 0) {
+    return [];
+  }
+
+  const changes = imports
+    .map(p => (p.importClause!.namedBindings! as ts.NamedImports).elements)
+    .reduce((imports, curr) => imports.concat(curr), [] as ts.ImportSpecifier[])
+    .map(specifier => {
+      if (!ts.isImportSpecifier(specifier)) {
```

Comment:
Is this not a duplicate of code in `ast-utils.ts`#L666

Reply:
Do you mean that this code is repeated in all of the modules? 
This is by design. If I'm not mistaken by doing it this way, we don't need to create a schematics-core package. (see #1025 and #1044).